linker/shared/src/main/scala/org/scalajs/linker/backend/javascript/SourceMapWriter.scala

```diff
@@ -207,7 +207,7 @@ final class SourceMapWriter(out: ByteArrayWriter, jsFileName: String,
 
   import SourceMapWriter._
 
-  private val sources = new ListBuffer[String]
+  private val sources = new ListBuffer[SourceFile]
```

Comment:
The title of this commit has a typo: "do doComplete" instead of "to doComplete".